public/app/features/dashboard-scene/settings/DeleteDashboardButton.tsx

```diff
@@ -80,8 +80,9 @@ export function DeleteDashboardModal({ dashboardTitle, onConfirm, onClose }: Del
             <>
               <Text element="p">
                 <Trans i18nKey="dashboard-settings.delete-modal-restore-dashboards-text">
-                  This action will mark the dashboard for deletion in 30 days. Your organization administrator can
```

Comment:
I think it would be best to leave as-is for now, until we have these changes implemented